modules/siddhi-core/src/main/java/io/siddhi/core/event/stream/MetaStreamEvent.java

```diff
@@ -40,6 +40,15 @@
     private StreamDefinition outputStreamDefinition;
     private EventType eventType = EventType.DEFAULT;
 
+    private void addOnAfterWindowData(Attribute attribute) {
```

Comment:
We should not force addOnAfterWindowData or  addBeforeWindowData. 
It should automatically happen based on the window initialization. 
Remove this methods.

Reply:
it is needed for cloning since I have to add the existing on after and on before window data to new cloned MetaStreamEvent